SecurityPkg/Library/AuthVariableLib/AuthService.c

```diff
@@ -439,7 +442,14 @@ CheckSignatureListFormat (
   // Walk through the input signature list and check the data format.
   // If any signature is incorrectly formed, the whole check will fail.
   //
-  while ((SigDataSize > 0) && (SigDataSize >= SigList->SignatureListSize)) {
+  // MU_CHANGE [START] - CodeQL change
+  while (SigDataSize > 0) {
+    SigListSize = (UINTN)SigList->SignatureListSize;

```

Comment:
nit: this seems fine, but the smaller change, like the two below, would be to just do the cast in the while loop, right?